extensions/amp-live-list/amp-live-list.md

```diff
@@ -79,7 +79,7 @@ In most implementations for live blogs, content is either pushed by the server
 to the client instance of a page, or the client polls a JSON endpoint to receive
 updates. The implementation for this component is different, in that the client
 instance of the page polls the server copy of the document for updates to the
-`items` reference point.
+`items` reference point. For instance: if the user is viewing a document served from an AMP cache, the client will poll that document hosted on that AMP cache for updates; if the user is viewing a document served from a web publisher's origin domain (e.g. "example.com"), then the client will poll the document hosted on that origin domain for updates.
 
```

Comment:
For the Google AMP Cache that's right — if the user is looking at a document served from the cache, each request (the initial page request, then each subsequent poll) will return content refreshed with the cache's one-behind (or stale-while-revalidate) model.

Web publishers can determine how fresh the content served is by:

a) using the Google AMP Cache's [update ping](https://developers.google.com/amp/cache/update-ping) when new content is published, in order to trigger the cache to fetch the newest version of the doc.

b) Tuning the polling frequency of the live blog to reduce the amount of time between the initial page load and the first poll

In practice, though, live blogs are typically made for breaking news, which may tend to get higher traffic (and inherently issue more page requests due to the polling mechanic), in turn triggering more page requests, and more frequent updates to caches using the one-behind model — effectively solving the issue itself.